src/plugins/vis_type_timeseries/public/application/components/vis_types/index.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+
+import { IUiSettingsClient } from 'src/core/public';
+import { PersistedState } from 'src/plugins/visualizations/public';
+
+/**
+ * TSVB visualizations are not typed yet.
+ */
+
+// @ts-expect-error
+import { TimeseriesVisualization as timeseries } from './timeseries/vis';
```

Comment:
what do you think about lazy loading of these components? I just think if the user sees only one type on the dashboard, why do we need to load all

Reply:
This is a good idea. I tried to implement this approach, but noticed that it creates a lot of small chunks.
So I have a concern, what is a better approach here..
You could take a look at 2 screenshots below:

- lazy load each visualization type, present couple of them in dashboard mode -> 13 small sized chunks for 4 vis types:

   ![image](https://user-images.githubusercontent.com/31325372/99689083-95684900-2a97-11eb-91ba-db4ed769ae11.png)

- current approach -> 2 chunks, but they are not so huge also

  ![image](https://user-images.githubusercontent.com/31325372/99689230-cc3e5f00-2a97-11eb-92d8-67b1eb7d3a9a.png)


So I wonder, is it worthy to do such a split?.. What are your thoughts?

Reply:
@stratoula  what do you think? I like idea with small chunks cause not sure that users use all tabs of TSVB  on dashboard. But I'm ok with both solutions.

Reply:
I like the idea too 🙂 I don't think that they load each TSVB chart type on a dashboard too. But as Daniil said the two chunks are not so big so I don't have strong objections here

Reply:
@sulemanof please do what you think is right =)

Reply:
@alexwizp @stratoula I'll pay more attention at the loading time of both variants in that case, and will post a comparison )

Reply:
Sounds great